### ISSUE TITLE
atmega328 encoder becomes unresponsive and does not recover

### ISSUE THREAD

**Issue by jqquanbeck** [SDSU-Robotics/srr_arduino#3]

**Comment by magicbycalvin** [comment]
It appears that it does not recover due to I2C issues. The first solution to the issue would be changing the request from the Raspberry Pi to six 4-byte requests instead of one 24-byte request. Conversely, one could change the slave write on the Arduino from six 4-byte frames to one 24-byte frame. The latter solution would be preferable since it avoids additional stop/start bits being sent across the line.

**Comment by jqquanbeck** [comment]
I agree, the latter sounds like a good solution. It will also avoid any issues where the order gets mixed up for the 6 values.
